src/main/java/com/epam/aidial/core/security/IdentityProvider.java

```diff
@@ -78,6 +91,17 @@ public IdentityProvider(JsonObject settings, Vertx vertx, Function<String, JwkPr
             jwkProvider = jwkProviderSupplier.apply(jwksUrl);
         }
 
+        try {
+            String userinfoEndpoint = settings.getString("userinfoEndpoint");
```

Comment:
please, update README.md to describe new IdP setting

Reply:
updated.